app/controllers/application_controller.rb

```diff
@@ -1,2 +1,7 @@
 class ApplicationController < ActionController::Base
+  protect_from_forgery with: :exception
+  before_action :current_user
+  def current_user
+    @current_user = User.first
```

Comment:
- [ ] Please remove the instance variable `@current_user` because when the method gets invoked we want the output of `User.first` directly. 🙏